Xit/FileView/CommitEntryController.swift

```diff
@@ -238,6 +260,21 @@ final class CommitEntryController: NSViewController,
     
     return bar
   }
+
+  func updateGuide()
+  {
+    if UserDefaults.xit.showGuide
```

Comment:
Use the `defaults` property instead of referencing `UserDefaults.xit` directly

Reply:
done!